package.json

```diff
@@ -638,6 +669,178 @@
                     "default": false
                   }
                 }
+              },
+              "pipeTransport": {
+                "type": "object",
```

Comment:
Attach with pipeTransport. There is a 'required' property that you should be able to add here to do that.